### PR TITLE
Refine Demagog integration after server is online

### PR DESCRIPTION
**Pull request by glowka**
- [x] Use `statement.attributes.sources` field instead of `statement.attributes.source`
- [x] Omit invalid urls from `sources_list/` 
- [x] Use  temporarily current date (`timezone.now()`) as annotation's creation date which for imported annotations is not necessarily true as there is no `date` field in demagog API
- [x] Strip `explanation` off html tags
- [x] Fix CSRF issues related to HTTP_REFERER (override referer for csrf checking as we are extension and expect requests to come from other hosts)

### CODE REVIEW

File: apps/annotation/authentication.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        extension background and content-scripts.
+        At the same time we use CSP and CSRF with the cookie accessible only on our website or by extension.
+        """
+        http_referer = request.META.get('HTTP_REFERER')
+        if 'HTTP_HOST' in request.META:
+            http_host = "https://%s/" % request.META['HTTP_HOST']
+            request.META = request.META.copy()
+            request.META['HTTP_REFERER'] = http_host
+        try:
+            super().enforce_csrf(request)
```
[the remark this answers is not in the document]
Comment (glowka) [reply apps/annotation/authentication.py]: Why would it? It just guarantees that referrer is not the reason of csrf failure, but there are other 5 checks that are still done.

[the remark this answers is not in the document]
Comment (kaglowka) [reply apps/annotation/authentication.py]: Ah, OK, I see, `enfore_csrf` is just an additional check on the DRF side.

[the remark this answers is not in the document]
Comment (glowka) [reply apps/annotation/authentication.py]: I am not sure what you mean, but in general DRF disables CSRF and if you really want to use it, you use This Authentication class that implements enforce_csrf which enbales it again. So I guess it is not exactly _additional_.

[the remark this answers is not in the document]
Comment (glowka) [reply apps/annotation/authentication.py]: OK I think I see the thing that might have misled you. *I am not using `except` here*, I am using `finally` which guarantees that no matter what happen this code is executed (to _clean_ after myself), but I do not interfere with exception propagation.

**Pull request merged**